strapi/strapi
# [Upgrade Tool] Add an option to specify the project path

### What does it do?

Allow configuring the Strapi project path from the CLI using either
- `-p <path>`
- or `--project-path <path>`

### Why is it needed?

Add more configuration capabilities to the upgrade tool CLI

### How to test it?

`yarn setup`
`cd examples`
`../packages/utils/upgrade/bin/upgrade.js next --dry-run` **// this should fail "no Strapi project found"**
`../packages/utils/upgrade/bin/upgrade.js next --dry-run -p getstarted` **// this should detect the getstarted project**
